sys/src/9/ufs/ufs/ufs_inode.c

```diff
@@ -47,7 +54,19 @@
 #endif
 #ifdef UFS_GJOURNAL
 #include <ufs/ufs/gjournal.h>
-#endif
+#endif*/
+
+
+/*
+ * Conversion tables for conversion from vnode types to inode formats
+ * and back.
+ */
+static Vtype iftovt_tab[16] = {
```

Comment:
#define	IFMT		0170000		/* Mask of file type. */
#define	IFIFO		0010000		/* Named pipe (fifo). */
#define	IFCHR		0020000		/* Character device. */
#define	IFDIR		0040000		/* Directory file. */
#define	IFBLK		0060000		/* Block device. */
#define	IFREG		0100000		/* Regular file. */
#define	IFLNK		0120000		/* Symbolic link. */
#define	IFSOCK		0140000		/* UNIX domain socket. */
#define	IFWHT		0160000		/* Whiteout. */

They have a few.  Hopefully we can trim it back a bit without losing UFS2 compatibility.